app/lib/.server/llm/model.ts

```diff
@@ -83,8 +84,11 @@ export function getHuggingFaceModel(apiKey: OptionalApiKey, model: string) {
 }
 
 export function getOllamaModel(baseURL: string, model: string) {
+  const savedCtx = Cookies.get('ollamaDefaultCtx');
+  const numCtx = savedCtx ? Number(savedCtx) : DEFAULT_NUM_CTX;
```

Comment:
this is server code, so js-cookie will not work here 

check api.chat.ts file to see how can you get the cookies data